Source/SwiftLintFramework/Helpers/Reachability.swift

```diff
@@ -0,0 +1,39 @@
+#if !os(Linux)
+import SystemConfiguration
+#endif
+
+/// Helper class providing the static helper method `isConnectedToNetwork()`
+internal class Reachability {
+    /// Returns whether the device is connected to a network, if known.
+    /// On Linux, this always evaluates to `nil`.
+    internal static func isConnectedToNetwork() -> Bool? {
```

Comment:
Maybe this instead?

```suggestion
    internal static func isConnectedToNetwork() -> ConnectivityStatus {
```

where

```swift
enum ConnectivityStatus { case connect, disconnected, unknown }
```